### PR TITLE
Cleaning project

### PR DESCRIPTION
**Pull request by marvol61**
Hello @rodi0878 , I just saw your program for creating Newton´s Fractal pictures.

It is very good, but I think that the code should be cleaner. Check it!

Thank you for you answer.

### CODE REVIEW

File: INPTPZ1/FractalModel.cs [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        public FractalModel()
+        {
+            roots = new List<ComplexNumber>();
+            colors = new Color[]
+            {
+                Color.Red, Color.Blue, Color.Green, Color.Yellow, Color.Orange, Color.Fuchsia, Color.Gold, Color.Cyan, Color.Magenta
+            };
+        }
+
+        public void GenerateNewtonFractal(Bitmap bitmap)
```
Comment (bLuma) [inline INPTPZ1/FractalModel.cs:26]: This method is still too long and do too much things at once.

File: INPTPZ1/ValueObjects/Polynome.cs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            ComplexNumbers = complexNumbers;
+        }
+
+        public Polynome Derive()
+        {
+            List<ComplexNumber> roots = new List<ComplexNumber>();
+
+            for (int i = 1; i < ComplexNumbers.Count; i++)
+            {
+                roots.Add(ComplexNumbers[i].Multiply(new ComplexNumber() { Re = i }));
```
Comment (bLuma) [inline INPTPZ1/ValueObjects/Polynome.cs:29]: What about some factory method... ComplexNumber.fromReal(i)

File: INPTPZ1/ValueObjects/Polynome.cs [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            }
+
+            return new Polynome(roots);
+        }
+        
+        public ComplexNumber Eval(ComplexNumber x)
+        {
+            ComplexNumber s = ComplexNumber.Zero;
```
Comment (bLuma) [inline INPTPZ1/ValueObjects/Polynome.cs:37]: s, coef, bx - are these names some cryptic mumbling?

**Review by bLuma — CHANGES_REQUESTED** [review]
There are still few things that would be nice to fix before merging...

File: INPTPZ1/ValueObjects/Polynome.cs (hunk 3 above)
Comment (marvol61) [reply INPTPZ1/ValueObjects/Polynome.cs:37]: Well, there is a lot of changes I´d like to do. But to be honest, in my view this topic(Newton´s Fractals) si not easy to understand. I wanted to changes these names, but I don´t know what these variables mean. As well as method name. Can I ask you for deeper explanation? Thank you.

Comment (bLuma) [reply INPTPZ1/ValueObjects/Polynome.cs:37]: You have a polynome (f(x) = 4 + 3x + 2xx) and you want to solve f(x) for x (specified by parameter). So...
"ComplexNumbers" is stupid name for attribute as these are coefficients of polynome.
coef - is specific coefficient (respectively its numeric value)
bx - is "x power to i", where i is current coefficient power 
s - f(x)

in last iteration you're solving "2xx"... coef = 2, bx = x*x, s += coef * bx ~ 2 * x*x

**Pull request closed without merging**